src/reactviews/pages/QueryResult/queryResultPane.tsx

```diff
@@ -226,7 +246,7 @@ export const QueryResultPane = () => {
                                 });
                             });
                     }}
-                    ref={gridRef}
+                    ref={(el) => (gridRefs.current[idx] = el!)}
```

Comment:
what's `el`?  might be worth using a longer name if the abbreviation isn't intuitive.